module/zfs/vdev_raidz_math.c

```diff
@@ -22,14 +22,14 @@
  * Copyright (C) 2016 Gvozden Nešković. All rights reserved.
  */
 
+#include <sys/simd.h>
 #include <sys/zfs_context.h>
 #include <sys/types.h>
 #include <sys/zio.h>
```

Comment:
1. `zfs_context.h`
2. `sys/types`
3. `sys/simd.h` maybe?

Reply:
zfs_context.h includes vnode.h and we need elf_common.h to be processed before vnode.h otherwise the hack does not work